repose-aggregator/commons/utilities/src/main/java/org/openrepose/commons/utils/servlet/http/MutableHttpServletResponse.java

```diff
@@ -302,11 +302,16 @@ public void sendError(int code) throws IOException {
 
     @Override
     public void sendError(int code, String message) {
-        super.setStatus(code);
-        this.message = message;
+        setStatus(code, message);
         error = true;
     }
 
+    @Override
+    public void setStatus(int code, String message) {
+        super.setStatus(code, message);
+        this.message = message;
```

Comment:
I hadn't been paying that much attention to this until now, but this is kind of a weird thing to do.

First, the double parameter setStatus method has been deprecated (due to the lack of parameter clarity). See: https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#setStatus(int, java.lang.String)

Second, we're now changing both the status message and the body of the response to be identical. That's kind of odd.

Third, sendError already has the correct behavior, I believe. Our sendError could just chain into super.sendError, store the message, and set the flag. Having our send error call super.setStatus(x, y) is changing the contracted behavior of sendError, which normally writes a body and not the status message. See:
https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#sendError(int, java.lang.String)

Have I mentioned how much I dislike our mutable wrapper jank? =P